src/test/java/uk/gov/companieshouse/web/pps/util/PenaltyUtilsTest.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.companieshouse.web.pps.util;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Please can you change your IntellJ Java settings for imports to steer away from using `.*`.
I must have bumped mine up to `999`.